daemon/src/taker_cfd.rs

```diff
@@ -184,18 +184,23 @@ impl Actor {
             .send(load_all_cfds(&mut conn).await?)?;
         let cfd = load_cfd_by_order_id(order_id, &mut conn).await?;
 
-        // let latest_announcement = self
-        //     .latest_announcement
-        //     .to_owned()
-        //     .context("Unaware of oracle's latest announcement.")?;
-
-        // self.oracle_actor
-        //     .do_send_async(oracle::MonitorEvent {
-        //         event_id: latest_announcement.id,
-        //     })
-        //     .await?;
+        let offer_announcements = self
+            .latest_announcements
+            .clone()
+            .context("No oracle announcements available")?;
+        let offer_announcement = offer_announcements
+            .0
+            .iter()
+            .find(|announcement| announcement.id == cfd.order.oracle_event_id)
+            .context("Order's announcement not found in list of current oracle announcements")?;
```

Comment:
👍 So instead of asking the oracle actor for a specific announcement we assume that the announcement in the order we get from the maker is within our `latest_announcements` state? It should work and it's definitely simpler to implement it this way. 

Perhaps we can go back an introduce a dedicated message later on.